From 2f36b16f55c88bb244f1aa8fc7d77d0c5df9c8e2 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Fri, 26 May 2017 12:59:40 +0200
Subject: [PATCH] Native DSD support for W4S DAC-2v2SE

This patch add native DSD support for the W4S DAC-2v2SE.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index d7b0b0a..982c31b 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1357,6 +1357,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
 	case USB_ID(0x20b1, 0x3023): /* Aune X1S 32BIT/384 DSD DAC */
 	case USB_ID(0x2616, 0x0106): /* PS Audio NuWave DAC */
+	case USB_ID(0x27f7, 0x3002): /* W4S DAC-2v2SE */
 		if (fp->altsetting == 3)
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
 		break;
-- 
2.9.4

